documentation/docs/operation/commands/history.md

```diff
@@ -81,4 +81,29 @@ Using the same example, get the starting quote value. Convert the starting ETH a
 ```
 Return % = -0.562 USDT / 2539.13 USDT
 Return % = -0.0222 %
+```
+
+
+## Net Delta
+
+Net Delta is calculated based on the difference of starting balance and current balance, while Trade Delta is tokens acquired after executing trades. Both Net Delta or Trade Delta can be positive or negative (profit or loss).
+
+$Net Delta = Starting - Current$
```

Comment:
To make it consistent across this page, use lower case just like the rest.

```suggestion
$net_delta = $starting_balance - $current_balance
```

Reply:
Noted, will revise, thanks.